### PR TITLE
 accessibility- unique gallery id 

### PR DESCRIPTION
**Pull request by yuvalab**
Using the same ID on more than one element may cause screen readers and other assistive technologies to only announce the first element with the shared ID, preventing users from accessing the later elements.

### CODE REVIEW

File: packages/gallery/tests/drivers/reactDriver.js [hunk 1]
```diff
@@ -178,6 +178,10 @@ class galleryDriver {
     };
   }
 
+  getContainer() {
+    return this.find.selector('#pro-gallery-container-default-dom-id');
```
Comment (nirnaor) [inline packages/gallery/tests/drivers/reactDriver.js]: @yuvalab 
What will happen here when the someone in the test will define an ID as a prop ?

**Review by nirnaor — CHANGES_REQUESTED** [review]

File: packages/gallery/tests/drivers/reactDriver.js (hunk 1 above)
Comment (yuvalab) [reply packages/gallery/tests/drivers/reactDriver.js]: you mean i need to let props.id  || the default string?

Comment (nirnaor) [reply packages/gallery/tests/drivers/reactDriver.js]: Yes

Comment (nirnaor) [reply packages/gallery/tests/drivers/reactDriver.js]: If the user decides in his test that he wants to use an `id` in props - the `getContainer` breaks

**Review by nirnaor — APPROVED** [review]

**Pull request merged**